deploy/deployUtils.ts

```diff
@@ -771,6 +771,118 @@ export async function stealFundsFromWhales(
   }
 }
 
+export async function deployCrossChainSystemOnSideChain(
+  hre: HardhatRuntimeEnvironment,
+) {
+  const { deployments, getNamedAccounts, ethers } = hre
+  const { get, execute, deploy, save } = deployments
+  const { deployer, crossChainDeployer } = await getNamedAccounts()
+
+  // set owner as deployer until all contracts are deployed
+  const owner = deployer
+
+  const xChainFactoryDeployOptions = {
```

Comment:
No `skipIfAlreadyDeployed: true` needed?

Reply:
Ah I had to remove that config to ensure fresh deployments. More details in this PR.
https://github.com/saddle-finance/saddle-contract/pull/750